frontend/src/pages/WorkloadDetails/WorkloadDetailsPage.tsx

```diff
@@ -322,8 +318,7 @@ class WorkloadDetailsPageComponent extends React.Component<WorkloadDetailsPagePr
           // Ztunnel doesn't have Envoy
           hasIstioSidecars =
             hasIstioSidecars ||
-            (!!pod.containers &&
-              pod.containers.some(cont => cont.name === istioProxyName && workload.ambient !== 'ztunnel'));
+            (!!pod.containers && pod.containers.some(cont => cont.name === istioProxyName && workload.isZtunnel));
```

Comment:
That should be !workload.isZtunnel, so the Envoy tab is hidden for ztunnel: 

![image](https://github.com/user-attachments/assets/5f1f49f5-8051-40c9-b8c6-83bab248c074)

Reply:
Nice catch, thanks!